apps/prairielearn/src/server.ts

```diff
@@ -2428,14 +2427,14 @@ if (esMain(import.meta) && config.startServer) {
           ssl: config.postgresqlSsl,
           errorOnUnusedParameters: config.devMode,
         };
-        function idleErrorHandler(err) {
+        function idleErrorHandler(err: Error, _client: sqldb.PoolClient) {
           logger.error('idle client error', err);
           Sentry.captureException(err, {
             level: 'fatal',
             tags: {
               // This may have been set by `sql-db.js`. We include this in the
```

Comment:
Nit, as long as we're changing things here:

```suggestion
              // This may have been set by `@prairielearn/postgres`. We include this in the
```